### PR TITLE
Add required tools and updatesegmenter

### PR DESCRIPTION
**Pull request by vincentmrg**

### CODE REVIEW

File: update_segmenter.py [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+            if  podready != "1/1" and podready != "2/2":
+                ready = False
+
+            # # Accelerate termination by sending die signal
+            # if get_pod_status(pod) == "Terminating":
+            #     if podready == "1/1" or podready == "2/2":
+            #         if pod.metadata.name not in died:
+            #             died.append(pod.metadata.name)
+            #             send_die_to_pod(pod)
+
```
Comment (TagadaPoe) [inline update_segmenter.py:361]: Unused code ?
```suggestion
```

File: update_segmenter.py [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+    if len(newconfs):
+        notused = False
+    else:
+        notused = True
+
+    # NOT NEEDED
+    # # Decativate conf in ainode
+    # for conf in newconfs:
+    #     put_ainode_conf(conf)
+
```
Comment (TagadaPoe) [inline update_segmenter.py:423]: Unused code ?
```suggestion
```

File: update_segmenter.py [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+    # Reset replicas to nominal value
+    if overbw is False or nbupstream != 1 or notused is True:
+        await put_deployment_replicas(deployment,nbreplicas)
+
+    # NOT NEEDED
+    # # Reactivate conf in ainode
+    # for conf in modifiedconfs:
+    #     put_ainode_conf(conf)
+
```
Comment (TagadaPoe) [inline update_segmenter.py:444]: Unused code ?
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -30,5 +30,50 @@ These files should  also follow a strict JSON format :
...
+### Usage
+
+- -h --help: Show this help message
+- -v VERSION, --version VERSION: The new segmenter version to update
+- -n NAME, --name NAME: Specify the basename of the segmenters
+- -d, --display: Display ongoing update
+- -u, --upgrade: Do upgrade
+- -o, --overbandwidth: Allow overbandwidth for mono segmenter
+- -p, --parallel: Allow parallel update of segmenters
+- -g GROUP [GROUP ...], --group GROUP [GROUP ...]: Specify the list of group to update
```
[the remark this answers is not in the document]
Comment (vincentmrg) [reply README.md:59]: @Thierry-35 ?

File: README.md [hunk 5]
```diff
@@ -30,5 +30,50 @@ These files should  also follow a strict JSON format :
 
 ### API
 
-Both can be used with the **Kubernetes API** or with the **external API**. By default, they use the kubernetes API, to use the external use the options `-A 
+Both can be used with the **Kubernetes API** or with the **external API**. By default, they use the kubernetes API, to use the external use the options `-A
 api.mycluster.com -u myuser:mypassword`.
+
+## Update scripts
+
+The script update_segmenter.py allows the massive update of segmenters unit.
```
[the remark this answers is not in the document]
Comment (vincentmrg) [reply README.md:38]: Thanks for the update !

File: update_segmenter.py (hunk 3 above)
Comment (vincentmrg) [reply update_segmenter.py:444]: @Thierry-35

**Pull request merged**